pkg/config/config.go

```diff
@@ -843,12 +843,14 @@ func InitConfig(config Config) {
 	// internal profiling
 	config.BindEnvAndSetDefault("internal_profiling.enabled", false)
 	config.BindEnv("internal_profiling.profile_dd_url")
+	config.BindEnvAndSetDefault("internal_profiling.unix_socket", "")
```

Comment:
One last request, can you add a comment here with an example value? Its unclear if it should be just the raw path to a unix socket vs a URL with the `unix://` scheme prepended.

Reply:
Done in [b5a52f0](https://github.com/DataDog/datadog-agent/pull/18544/commits/b5a52f0312575af3c7861e8c3dcd9f86f2b6c591)